### ISSUE TITLE
Is The Mod Abandoned?

### ISSUE THREAD

**Issue by Weather-OS** [Nukem9/dlssg-to-fsr3#412]

**Comment by Sousaboi** [comment]
Pretty much….Sadly…

**Comment by TimmyP777** [comment]
Yes. Its long gone. Nvidia was here. Nvidia is on every single post about it on reddit, GN videos, DF, videos... and the comments are removed. This mod was deliberately held back, so people could not gain the valuable understanding of what it does.

What I want to know is how they are deleting the posts so fast? GN\DF they are gone in less the a minute. Reddit you can  only post about this mod if you are lucky.

They had a "dedicated" thread, but its actually there to consolidate.

Dont believe me? Test it. This **** is so corrupt and widespread. Roaches are literally everywhere.
